src/api.js

```diff
@@ -60,11 +61,11 @@ export default class Api {
    * @return {String}
    */
   endpoints(options={fullUrl: true}) {
-    var root = options.fullUrl ? this.remote : `/${this.version}`;
+    var root = options.fullUrl ? `${this.remote}/` : `/${this.version}/`;
     var urls = {
       root:                   () => root,
-      batch:                  () => `${root}/batch`,
-      bucket:           (bucket) => `${root}/buckets/${bucket}`,
+      batch:                  () => `${root}batch`,
```

Comment:
Why don't you keep the slashes there? I think it best to keep all prefixes starting with a ``/``